settings: Bump Facebook Graph API version to v19.0

dbnicholson opened the pull request:
Per the Graph API changelog[1], v19.0 is the current version. The only Graph API used directly by allauth is the User Picture API[2], and that does not appear to have changed. The actual login calls are managed by the Facebook JS SDK, which presumably knows how to use the newer Graph API version.

1. https://developers.facebook.com/docs/graph-api/changelog/
2. https://developers.facebook.com/docs/graph-api/reference/user/picture/

https://phabricator.endlessm.com/T35222

dbnicholson committed:
settings: Bump Facebook Graph API version to v19.0

Per the Graph API changelog[1], v19.0 is the current version. The only
Graph API used directly by allauth is the User Picture API[2], and that
does not appear to have changed. The actual login calls are managed by
the Facebook JS SDK, which presumably knows how to use the newer Graph
API version.

1. https://developers.facebook.com/docs/graph-api/changelog/
2. https://developers.facebook.com/docs/graph-api/reference/user/picture/

https://phabricator.endlessm.com/T35222

wjt reviewed (approved):
This change looks good but sadly the PR failed to build for Vault Reasons

dbnicholson commented:
> This change looks good but sadly the PR failed to build for Vault Reasons

Right. The [official Vault image](https://hub.docker.com/_/vault) is deprecated and instead you're supposed to use the [hashicorp/vault image](https://hub.docker.com/r/hashicorp/vault). Lemme fix that...

dbnicholson committed:
Dockerfile: Use hashicorp/vault image

The official Vault image[1] is deprecated and instead you're supposed to
use the Hashicorp Vault image[2]. While here, prefix the image repos
with `docker.io` so the image can be built with `podman`, too.

1. https://hub.docker.com/_/vault
2. https://hub.docker.com/r/hashicorp/vault

https://phabricator.endlessm.com/T35222
